x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/waterfall/waterfall_chart_wrapper.tsx

```diff
@@ -77,6 +82,14 @@ export const WaterfallChartWrapper: React.FC<Props> = ({ data }) => {
         tickFormat={(d: number) => `${Number(d).toFixed(0)} ms`}
         domain={domain}
         barStyleAccessor={(datum) => {
+          if (SAFE_PALETTE[6] === datum.datum.config.colour) {
+            return {
+              rect: {
+                fill: datum.datum.config.colour,
+                opacity: '0.15',
```

Comment:
Can we pull this value out to a named constant above this function? `QUEUED_OR_BLOCKED_OPACITY` or something to that effect.